github-config.yaml

```diff
@@ -333,6 +336,50 @@ orgs:
         privacy: closed
         repos:
           ai-for-cloud-ops: admin
+      blog:
+        description: Writers and reviewers of Op1st blog posts
+        privacy: closed  # Keep as closed, more info: https://docs.github.com/en/rest/reference/teams#create-a-team
+        members:
+          - jeremyeder

```

Comment:
nit - is this list intended to be alphabetical?

Reply:
```suggestion
          - jeremyeder
          - oindrillac
          - aakankshaduggal
```

Reply:
> nit - is this list intended to be alphabetical?

Yes, and not have double Jeremy entries :)

These should both be fixed/addressed in ff2f498